guides/source/layouts_and_rendering.md

```diff
@@ -277,6 +277,16 @@ since an attacker could use this action to access security sensitive files in yo
 
 TIP: `send_file` is often a faster and better option if a layout isn't required.
 
+#### Rendering view components
```

Comment:
Honestly don't think this is an apt description. The support is for any object that responds to `render_in`, the fact that someone can implement components based on that is a side effect.